products/BellHybrid/apps/application-bell-onboarding/windows/OnBoardingInstructionPromptWindow.cpp

```diff
@@ -27,10 +27,7 @@ namespace gui
 
     bool OnBoardingInstructionPromptWindow::onInput(const InputEvent &inputEvent)
```

Comment:
hmm so remove it if it is not used?

Reply:
we need to override it so it does nothing